firebase-app/src/androidInstrumentedTest/kotlin/dev/gitlive/firebase/firebase.kt

```diff
@@ -6,8 +6,7 @@
 package dev.gitlive.firebase
 
 import androidx.test.platform.app.InstrumentationRegistry
-import kotlinx.coroutines.runBlocking
 
 actual val context: Any = InstrumentationRegistry.getInstrumentation().targetContext
 
-actual fun runTest(test: suspend () -> Unit) = runBlocking { test() }
+actual fun runTest(test: suspend () -> Unit) = kotlinx.coroutines.test.runTest { test() }
```

Comment:
so now all the delays are skipped in tests? I wonder whether it has a big impact